packages/codegen-ui-react/lib/utils/constants.ts

```diff
@@ -22,3 +22,49 @@ export const STORAGE_FILE_ALGO_TYPE = 'SHA-1';
 export const AMPLIFY_JS_V5 = '5.0.0';
 
 export const AMPLIFY_JS_V6 = '6.0.0';
+
+export const scriptingPatterns = [
+  // JavaScript functions
+  /eval\s*\(/i,
+  /Function\s*\(/i,
+  /setTimeout\s*\(/i,
+  /setInterval\s*\(/i,
+  /new\s+Function/i,
+  /import\s*\(/i,
+  /require\s*\(/i,
+
+  // DOM manipulation
+  /document\./i,
+  /window\./i,
+  /location\./i,
+  /localStorage\./i,
```

Comment:
This may break existing customers. There is a legitimate use case to access localStorage and sessionStorage in a binding property